packages/cactus-common/src/test/typescript/unit/js-object-signer.test.ts

```diff
@@ -1,5 +1,5 @@
-import test, { Test } from "tape";
-
+import "jest-extended";
+// import { RuntimeError } from "run-time-error";
```

Comment:
@awadhana Sorry for the nit-picking as usual. Please remove the commented line if it's no longer needed. Looks good otherwise though so we are almost there!

Reply:
@petermetz not a problem at all :) Clean code is the best code.

Reply:
@awadhana Yes, now we are cooking!